lib/WorseReflection/Core/Reflection/Collection/ClassLikeReflectionMemberCollection.php

```diff
@@ -68,9 +69,11 @@ final class ClassLikeReflectionMemberCollection extends AbstractReflectionCollec
 
     public static function fromClassMemberDeclarations(
         ServiceLocator $serviceLocator,
-        ClassDeclaration $class,
+        ClassDeclaration|ObjectCreationExpression $class,
         ReflectionClass $reflectionClass
     ): self {
+        assert($class->classMembers !== null, 'ObjectCreationExpression does not contain anonymous class');
```

Comment:
what could it contain? should be an exception if it should never happen (and it probably can) or the error needs to be handled without crashing (maybe via. `NotFound` exception)

Reply:
Yes, should probably assert that this is a `ClassMembers` variable. This happens when the code comes across an ObjectCreationExpression that does not contain an anonymous class.